config/application.properties.example

```diff
@@ -22,4 +22,9 @@ flyway.password=flyway
 flyway.driverClassName=org.postgresql.Driver
 flyway.location=classpath:db/migration
 
-spring.jpa.hibernate.ddl-auto
\ No newline at end of file
+spring.jpa.hibernate.ddl-auto
+
+smtp.host=127.0.0.1
+smtp.port=8888
+smtp.from.address=mail.formplayer.commcarehq.org
+smtp.to.address=exceptions@commcarehq.org
```

Comment:
we're going to need to be able to authentication (provide a username and password) in order for us to actually use our mail service. when we add it, it should still work for users that don't need to email